File: Mensuration/Triangle and Prism/Volume/VoP.java
import java.util.Scanner;
public class AoT {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 1st Side : ");
        int a=sc.nextInt();
        System.out.print("Enter 2nd Side : ");
        int b=sc.nextInt();
        System.out.print("Enter 3rd Side : ");
        int c=sc.nextInt();
        System.out.print("Enter Height : ");
        int h=sc.nextInt();
        sc.close();
        double s=(a+b+c)/2;
        double t=((s*(s-a))*(s-b))*(s-c);
        double vol=h*Math.sqrt(t);
        System.out.print("Volume : "+vol);
    }    
}

File: Mensuration/Triangle and Prism/Volume/VoP.c
#include<stdio.h>
#include<math.h>
void main(){
    int a,b,c,h;
    double s,t,vol;
    printf("Enter 1st Side : ");
    scanf("%d",&a);
    printf("Enter 2nd Side : ");
    scanf("%d",&b);
    printf("Enter 3rd Side : ");
    scanf("%d",&c);
    printf("Enter Height : ");
    scanf("%d",&h);
    s=(a+b+c)/2;
    t=s*(s-a)*(s-b)*(s-c);
    vol=h*sqrt(t);
    printf("Volume : %0.2f",vol);
}